code/modules/atmospherics/machinery/components/unary_devices/bluespace_sender.dm

```diff
@@ -170,8 +170,9 @@
 			. = TRUE
 
 		if("retrieve")
-			if(bluespace_network.total_moles() > 0)
-				var/datum/gas_mixture/remove = bluespace_network.remove(bluespace_network.total_moles())
+			var/total_moles = bluespace_network.total_moles()
+			if(total_moles > 0)
+				var/datum/gas_mixture/remove = bluespace_network.remove(total_moles)
```

Comment:
also should just be remove ratio (1), no need to cache

Reply:
Not sure how I missed that.